tensorflow/contrib/image/python/ops/image_ops.py

```diff
@@ -81,6 +81,31 @@ def rotate(images, angles, interpolation="NEAREST"):
     return output
 
 
+def translate(images, translations, interpolation="NEAREST"):
```

Comment:
name= arg on any function that creates new ops., use it inside a "with name_scope(...):" before creating any ops.  here, elsewhere.

Reply:
Added name= args and tf.name_scope(name) throughout.